cmake/templates/BoostConfig.cmake.in

```diff
@@ -150,3 +150,10 @@ endif()
 if(TARGET Boost::chrono)
   hunter_add_rt_library(Boost::chrono)
 endif()
+
+if(TARGET Boost::log AND "@BUILD_SHARED_LIBS@")
```

Comment:
Since we are changing internal Boost files please update `PACKAGE_INTERNAL_DEPS_ID` of all files from `cmake/projects/Boost` directory. See:
* https://docs.hunter.sh/en/latest/reference/user-modules/hunter_download.html

Reply:
I updated all `PACKAGE_INTERNAL_DEPS_ID` in `cmake/projects/Boost`